plugins/Login/stylesheets/login.less

```diff
@@ -108,3 +108,67 @@
         color: gray;
     }
 }
+
+.onboarding {
+    background: white;
+
+    h1 {
+        margin-top: 4rem;
+        margin-bottom: 1.5rem;
+        line-height: 24px;
+    }
+
+    .description {
+        margin-bottom: 1.5rem;
+        line-height: 24px;
+    }
+
+    label {
+        font-weight: 600;
+        color: unset !important;
+    }
+
+    .footerLinks {
```

Comment:
Is there any element having this class? Couldn't find one...

Reply:
Removed, I would have added for the additional 2 screens which we moved to CLOUD to now